roadmap.md

```diff
@@ -0,0 +1,34 @@
+# Project Roadmap
+
+This is where we‘ll define a few things about the library‘s goals.
+
+We haven‘t filled this out yet though. Care to help? See [`contributing.md`](./contributing.md)
+
+
+> Notice: sub-packages have their own `roadmap.md` files.
```

Comment:
It would be usefull to provide links, so it can be easily accessed.
Or it could be actually merged into one file as I would consider this as a one project, so the Project Management should be joined

Reply:
I understand your point, I will try to clean up and do everything in the main `roadmap.md` with sections for each package.